### PR TITLE
feat add signin page

### PR DESCRIPTION
**Pull request by WhiteNik16**

### CODE REVIEW

File: src/pages/SignInPage/SignInPage.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <Icon
+          width={88}
+          height={88}
+          className='sign-in-page__icon'
+          name={IconNames.User}
+        />
+        <span className='sign-in-page__title'>{'Sign in'}</span>
+        <p className='sign-in-page__description'>
+          {
+            'Manage your identity credentials and Soulbound Tokens (SBTs) easily from this dashboard'
```
Comment (napalmpapalam) [inline src/pages/SignInPage/SignInPage.tsx:39]: why no i18n?

Comment (napalmpapalam) [inline src/pages/SignInPage/SignInPage.tsx:39]: why no i18n?

File: src/pages/SignInPage/styles.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.sign-in-page {
+    display: flex;
+  align-items: center;
```
Comment (napalmpapalam) [inline src/pages/SignInPage/styles.scss:3]: !

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: src/pages/SignInPage/styles.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (WhiteNik16) [reply src/pages/SignInPage/styles.scss:3]: fixed

Comment (WhiteNik16) [reply src/pages/SignInPage/styles.scss:3]: fixed

**Pull request merged**